Woof/WoofTests/Model/KeyValueStorage/TestData/KeyValueStorage+TestKey.swift

```diff
@@ -0,0 +1,16 @@
+extension KeyValueStorage {
+    /// Keys in the key-value storage to use in unit tests.
+    enum TestKey: String, CaseIterable {
+        /// An example of key to use in the key-value storage.
+        case keyA = "A"
+
+        /// An example of key to use in the key-value storage.
+        case keyB = "B"
+
+        /// An example of key to use in the key-value storage.
+        case keyC = "C"
+
```

Comment:
nit: I didn't find the tests where we use it.

Reply:
Removed, thanks!